bot/settings.py

```diff
@@ -1,14 +1,13 @@
-import os
+from pathlib import Path
 
 from dotenv import load_dotenv
 from pydantic import Field
 from pydantic_settings import BaseSettings
 
-env_path: str = os.path.join(os.path.dirname(__file__), "..", ".env")
-if os.path.exists(env_path):
+env_path: Path = Path(__file__).parent.parent / ".env"
```

Comment:
no nie wiem nie podoba mi sie cos tutaj, czemu hardkodujesz sciezke z parent parent zamiast przekazac enva ktory ma wskazywac dotenva albo w ogole od razu wszystkie zmienne przekazac w jakims dockerfile/compose yaml?

Reply:
w sensie z tym przekazywaniem od razu to jest tego kwestia że ten .env jest tylko devowy normalnie to przekazuje wiesz jako zmienne w kontenerze więc na produkcji go nie ma

Reply:
Dobra to weź dorzuć log WARNING 

`Using dotenv file`

Żeby było wiadomo że jest wykorzystany i będzie git cały mr